instrument/IN6_Definition.xml

```diff
@@ -1426,10 +1426,10 @@ last-modified="2018-03-27 11:41:29">
   <!-- Detector tube shape. Cuboids 32 x 16 mm, 300 mm long -->
```

Comment:
The new shape has 31 mm along x. The old one is a bit skewed, the new shape is straight. Does the above comment need updating?

Reply:
Nice catch! The comment is indeed wrong. We need to take the 0.5mm thick walls into account in the x direction as well.